package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "function",
+  "version": "1.0.0",
+  "description": "aqa task",
+  "main": "index.js",
+  "scripts": {
+    "test": ""
```

```suggestion
    "test": "cypress run"
```

Add here command how run tests. And after that u will run the tests `npm run test`